crates/ide/src/folding_ranges.rs

```diff
@@ -418,4 +444,24 @@ fn foo<fold arglist>(
 "#,
         )
     }
+
+    #[test]
+    fn fold_region() {
+        log_init_for_test_debug();
+        // only error level log is printed on the terminal
+        log::error!("test fold_region");
```

Comment:
I don't think the logging stuff is needed(including the `env_logger` dev-dependency)?

Reply:
No, logging is not needed. I just wanted to have it temporarily.